### PR TITLE
feat: allow users to specify host and port while launching server

### PR DESCRIPTION
**Pull request by gaurav274**
1. Allow users to specify hostname and port while launching the server
2. Change the default port to 8803 to avoid conflict with Postgres.

### CODE REVIEW

File: script/test/package.sh [hunk 1]
```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
 
+## backward compatibility
+EVA_VERSION=$(pip show evadb | grep Version | awk '{print $2}')
+if [[ "$EVA_VERSION" < "0.2.4" ]]; then
```
Comment (jarulraj) [inline script/test/package.sh:5]: Good decision

File: eva/server/server.py [hunk 2]
```diff
@@ -38,6 +38,7 @@ async def start_eva_server(self, host: string, port: int):
         port: port of the server
         """
         logger.info("Start Server")
+        print(f"EVA Server started at host {host} and port {port}", flush=True)
```
Comment (jarulraj) [inline eva/server/server.py:41]: Is there a reason for using `print` instead of logger here?

File: eva/server/interpreter.py [hunk 3]
```diff
@@ -56,9 +56,8 @@ async def read_from_client_and_send_to_server(
     stdin_reader: StreamReader, writer: StreamWriter, server_reader: StreamReader
 ):
     VERSION = VERSION_DICT["VERSION"]
-    intro = "eva (v " + VERSION + ')\nType "EXIT;" to exit the client' + "\n"
-    sys.stdout.write(intro)
-    sys.stdout.flush()
+    intro = f"eva (v{VERSION})\nType 'EXIT;' to exit the client \n"
+    print(intro, flush=True)
```
Comment (jarulraj) [inline eva/server/interpreter.py:60]: Is there a reason for using `print` instead of logger or sys.stdout here?

**Review by jarulraj — APPROVED** [review]

**Comment by jarulraj** [comment]
Can we add a test case as well -- https://coveralls.io/builds/59972132/source?filename=eva%2Feva_cmd_client.py#L49

File: eva/server/server.py (hunk 2 above)
Comment (gaurav274) [reply eva/server/server.py:41]: Logger wasn't displaying anything. Maybe we have to set the level to `info`, wasn't sure if we want to do it.

File: eva/server/interpreter.py (hunk 3 above)
Comment (gaurav274) [reply eva/server/interpreter.py:60]: `print` is just a wrapper around `sys.stdout`.

**Review by xzdandy — APPROVED** [review]

**Pull request merged**